src/material/core/style/_focus-indicators.scss

```diff
@@ -0,0 +1,68 @@
+/// Mixin that turns on strong focus indicators.
+///
+/// @example
+///   .my-app {
+///     @include mat-strong-focus-indicators($config);
+///   }
+
+// Private sass variables that will be used as reference throughout component stylesheets.
+$private-default-focus-indicator-border-width: 3px;
+$private-default-focus-indicator-border-style: solid;
+$private-default-focus-indicator-border-color: transparent;
+$private-default-focus-indicator-border-radius: 4px;
+
+// Mixin that renders the focus indicator structural styles.
+@mixin private-structural-styling($prefix) {
+  .#{$prefix}-focus-indicator {
+    position: relative;
+
+    // Render the focus indicator on focus. Defining a pseudo element's
+    // content will cause it to render.
+    &::before {
+      @include layout-common.fill();
+      box-sizing: border-box;
+      pointer-events: none;
+      border: var(
+          --#{$prefix}-focus-indicator-border-width,
+          $private-default-focus-indicator-border-width
+        )
+        var(
+          --#{$prefix}-focus-indicator-border-style,
+          $private-default-focus-indicator-border-style
+        )
+        var(
+          --#{$prefix}-focus-indicator-border-color,
+          $private-default-focus-indicator-border-color
+        );
+      border-radius: var(
+        --#{$prefix}-focus-indicator-border-radius,
+        $private-default-focus-indicator-border-radius
+      );
+    }
+
+    // For all other components, render the focus indicator on focus.
```

Comment:
The phrase `For all other components,` no longer makes sense, because there are no comments listed in this file any more. Can we change this comment to: "By default, render the focus indicator when the focus indicator host element takes focus."?